test/cpp/jit/test_argument_spec.cpp

```diff
@@ -98,36 +98,6 @@ size_t hashCode(const TensorTypePtr& ptr) {
   return std::hash<TensorType>()(*ptr.get());
 }
 
-void testProfiledTensorTypeHashing() {
```

Comment:
Did this move, or does the just remove the hashing test? Seems like an important test.

Reply:
Forgot to port it. Also changed the tests that use `TensorType::create` in a way we actually use it.